src/modules/NtpClient.rb

```diff
@@ -353,18 +366,103 @@ def Write
     # @return [Boolean] True on success
     def Import(settings)
       log.info "Import with #{settings}"
-      # TODO: implement for chrony
+
+      unsupported  = UNSUPPORTED_AUTOYAST_OPTIONS.select { |o| settings.key?(o) }
+      if !unsupported.empty?
+        Yast::Report.Error(
+          format(
+            # TRANSLATORS: error report. %s stands unsuported keys.
+            _("Old unsupported profile detected. Ntp is not configured. Unsupported keys: '%s'."),
```

Comment:
"Ignoring the NTP configuration. The profile format has changed in an incompatible way. These keys are no longer supported: ..."

Reply:
👍